azure-local/concepts/compare-windows-server.md

```diff
@@ -69,60 +69,65 @@ The following table compares the workloads and benefits of Azure Local and Windo
+ <sup>1</sup> [AKS on Windows Server End of Support 27th March 2027](https://learn.microsoft.com/en-us/azure/aks/aksarc/aks-windows-server-retirement?tabs=ws).<br>
+ <sup>2</sup> Requires purchasing an Extended Security Updates (ESU) license key and manually applying it to every VM.
  
 ## Compare technical features
 
-The following table compares the technical features of Azure Local and Windows Server 2022.
-
-| Attribute | Azure Local | Windows Server 2022 |
-| ------------- | ------------------- | ----------------------- |
-| Hyper-V | Yes | Yes |
-| Storage Spaces Direct | Yes | Yes |
-| Software-Defined Networking | Yes | Yes |
-| Adjustable storage repair speed | Yes | Yes|
-| Secured-core Server| Yes | Yes |
-| Stronger, faster network encryption | Yes | Yes |
-| 4-5x faster Storage Spaces repairs | Yes | Yes |
-| Stretch clustering for disaster recovery with Storage Spaces Direct<sup>1</sup> | Yes | No |
-| High availability for GPU workload | Yes | No |
-| Restart up to 10x faster with kernel-only restarts | Yes | No |
-| Simplified host networking with Network ATC | Yes | No |
-| Storage Spaces Direct on a single machine | Yes | No |
-| Storage Spaces Direct thin provisioning | Yes | No |
-| Dynamic processor compatibility mode| Yes | No |
-| Cluster-Aware OS feature update | Yes | No |
-| Integrated driver and firmware updates | Yes (Integrated Systems only) | No |
-
-<sup>1</sup> Functionality only available in Azure Local, version 22H2.
+The following table compares the technical features of Azure Local and Windows Server 2025.
+
+| Attribute | Azure Local 23H2 / 24H2 | Windows Server 2025 | Windows Server 2022
+| ------------- | ------------------- | ----------------------- | ----------------------- |
+| Hyper-V | Yes | Yes | Yes |
+| Storage Spaces Direct | Yes | Yes | Yes |
+| Software-Defined Networking | Yes | Yes | Yes |
+| Adjustable storage repair speed | Yes | Yes | Yes |
+| Secured-core Server| Yes | Yes | Yes |
+| Stronger, faster network encryption | Yes | Yes | Yes |
+| 4-5x faster Storage Spaces repairs (256kb slab size) | Yes | Yes | Yes |
+| Stretch clustering for disaster recovery with Storage Spaces Direct | No <sup>1</sup> | Work in Progress | No |
+| High availability for GPU workload | Yes | Yes | No |
+| Restart up to 10x faster with kernel-only restarts | Yes <sup>2</sup> | No | No |
+| Simplified host networking with Network ATC | Yes | Yes | No |
+| Storage Spaces Direct on a single machine | Yes | Yes | No |
+| Storage Spaces Direct thin provisioning | Yes | Yes | No |
+| Dynamic processor compatibility mode| Yes | Yes | No |
+| Cluster-Aware OS feature update | Yes | Yes | No |
+| Integrated driver and firmware updates | Yes <sup>2</sup> | No | No
+
+<sup>1</sup> Functionality only available in Azure Local 22H2. These may be upgraded to supported versions with known feature limitations.<br>
+<sup>2</sup> With Azure Local Integrated Systems and Premier Systems Hardware only.
```

Comment:
moved to sup 2 for improved readability as in fact both features are bound to specific hardware requirements.